### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sagarnasit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,132 @@
-# EasyEngine/cron-command
+easyengine/cron-command
```
Comment (mbtamuli) [inline README.md]: ```# EasyEngine/cron-command```

It should be this syntax

File: README.md [hunk 2]
```diff
@@ -1,100 +1,132 @@
...
+We appreciate you taking the initiative to contribute to this project.
+
+Contributing isn’t limited to just code. We encourage you to contribute in the way that best fits your abilities, by writing tutorials, giving a demo at your local meetup, helping other users with their support questions, or revising our documentation.
+
+
+### Reporting a bug
+
+Think you’ve found a bug? We’d love for you to help us get it fixed.
+
+Before you create a new issue, you should [search existing issues](https://github.com/easyengine/cron-command/issues?q=label%3Abug%20) to see if there’s an existing resolution to it, or if it’s already been fixed in a newer version.
```
Comment (mbtamuli) [inline README.md]: Nitpick here but change `https://github.com/easyengine` to `https://github.com/EasyEngine`

File: README.md (hunk 1 above)
Comment (sagarnasit) [reply README.md]: This comes from `package_name` defined in composer.json file.

File: README.md (hunk 2 above)
Comment (sagarnasit) [reply README.md]: same as above.

**Pull request merged**